docker/app/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.4.2
```

Comment:
Try `alpine` images. They are small compared to default image. Same for `nginx`

Reply:
will take a look Thanks